openassessment/templates/openassessmentblock/edit/oa_edit_criterion.html

```diff
@@ -2,7 +2,7 @@
 {% spaceless %}
 <li class="openassessment_criterion is-collapsible" data-criterion="{{ criterion_name }}">
     <div class="openassessment_container_header openassessment_criterion_header view-outline">
-        <a class="action expand-collapse collapse"><i class="icon-caret-down ui-toggle-expansion"></i></a>
```

Comment:
Note to reviewers: this icon missed the fa conversion. When I changed it the icon did appear, but the functionality did not work (the section did not collapse when the icon or title beside it was clicked on). I created a bug to fix this if product desires, and am removing the icon so it does not look confusing.

https://openedx.atlassian.net/browse/TNL-3799